WordPress-VIP-Go/ruleset.xml

```diff
@@ -210,10 +210,7 @@
 		<!-- We trust that translations are safe on VIP Go -->
 		<severity>1</severity>
 	</rule>
-	<rule ref="WordPress.WP.GlobalVariablesOverride.Prohibited">
-		<!-- This is often a false positive. Still nice to flag for a check -->
-		<severity>3</severity>
-	</rule>
+	<rule ref="WordPress.WP.GlobalVariablesOverride.Prohibited"/>
```

Comment:
This line is not needed. The sniff (including this error code) is inherited from `WordPressVIPMinimum`, so doesn't have to be included explicitly.

Only reason to still have it, would be if the property I mentioned in my comment would be added: https://github.com/Automattic/VIP-Coding-Standards/pull/610#issuecomment-790186115

Reply:
I think I'll keep the property disabled for the time being since it seems to be discouraged per:
> Unfortunately, at this time, it is not possible to set this property for only a select group of files. With that in mind, setting this property is discouraged.
If we receive reports of it being an issue, we can re-visit later I'm thinking.